### PR TITLE
Add retry for aqa-tests repo

### PR DESCRIPTION
**Pull request by renfeiw**
Signed-off-by: renfeiw <renfeiw@ca.ibm.com>

### CODE REVIEW

**Comment by renfeiw** [comment]
testing failure case: Grinder/25475
success case: Grinder/25477

File: buildenv/jenkins/openjdk_tests [hunk 1]
```diff
@@ -275,21 +275,39 @@ def runTest() {
             def SCM_GIT_REPO_VAL = sh(script: "echo ${SCM_GIT_REPO}", returnStdout: true).trim()
             SCM_GIT_REPO_VAL = SCM_GIT_REPO_VAL.replace("https://github.com/","git@github.com:")
 
-            sh "git clone -b ${SCM_GIT_BRANCH} ${SCM_GIT_REPO_VAL} aqa-tests"
+            def count = 0
+            retry(5) {
+                if (count > 0) {
+                    def sleep_time = 300
+                    println("Sleep ${sleep_time} secs, then retry ${count}...")
+                    sh 'sleep 300'
```
Comment (llxia) [inline buildenv/jenkins/openjdk_tests:283]: We can directly `sleep(<SECONDS>)` https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#sleep-sleep

Comment (renfeiw) [reply buildenv/jenkins/openjdk_tests]: Updated

**Comment by renfeiw** [comment]
testing failure case: Grinder/25480
success case: Grinder/25481, 25482

File: buildenv/jenkins/openjdk_tests [hunk 2]
```diff
@@ -275,21 +277,35 @@ def runTest() {
             def SCM_GIT_REPO_VAL = sh(script: "echo ${SCM_GIT_REPO}", returnStdout: true).trim()
             SCM_GIT_REPO_VAL = SCM_GIT_REPO_VAL.replace("https://github.com/","git@github.com:")
 
-            sh "git clone -b ${SCM_GIT_BRANCH} ${SCM_GIT_REPO_VAL} aqa-tests"
+            retry(5) {
+                if (retry_count > 0) {
+                    println("Sleep ${sleep_time} secs, then retry ${retry_count}...")
```
Comment (llxia) [inline buildenv/jenkins/openjdk_tests:282]: `retry` and `sleep` already print msg (see below). I think we can remove the extra `println`.
```
Retrying
[Pipeline] {
[Pipeline] echo
Sleep 180 secs, then retry 4...
[Pipeline] sleep
Sleeping for 3 min 0 sec
 > git rev-parse --resolve-
````

Comment (renfeiw) [reply buildenv/jenkins/openjdk_tests]: updated

**Review by smlambert — APPROVED** [review]
I don't have visibility of the Grinders, but the changes LGTM, thanks!

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**